app/Http/Controllers/AttendanceController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Classes\Stream;
+use App\Models\Attendances\Attendance;
+
+class AttendanceController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware('auth');
```

Comment:
btw this is being taken care of in the `web.php` so adding it in the controller has no effect

Reply:
Sawa